css/single.project.css

```
@@ -339,14 +339,14 @@ ul, li {
     font-family: 'Russo One', sans-serif;
     font-size: 2.625rem;
     font-weight: 100;
-    color: #ffffff;
+    color: var(--text-light);
 }
 
 .project-main-info h4 {
     margin-top: 10px;
     font-family: 'Inter', sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)